api/src/main/resources/db/changelog/db.changelog-refdata.yaml

```diff
@@ -139,3 +139,15 @@ databaseChangeLog:
                 WHERE name = 'bar card';
                 INSERT INTO payment_provider (name, description)
                 VALUES ('bar card', 'Bar card');
+
+  - changeSet:
```

Comment:
There are few other changes requested in the story description. Are you doing as part of this PR?

Reply:
the story description is not completely valid. discussed with Kemi and Salman before doing this change.